### ISSUE TITLE
A change in Qt 5.11_beta3 leads to errors

### ISSUE THREAD

**Issue by vanyasem** [ubports/ubuntu-download-manager#4]
Arch GNU/Linux, `bionic` branch
```
CMake Error at src/downloads/qml/CMakeLists.txt:33 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".
```

**Comment by vanyasem** [comment]
According to [the official Qt documentation](https://doc.qt.io/qt-5/cmake-manual.html) `qt5_use_modules` macro is obsolete

**Comment by vanyasem** [comment]
Fixed by #5 on `bionic` branch. This should probably get backported to `xenial`. Or we should get rid of all other branches and leave only "master" and "stable" (as Rodney suggests) @mariogrip

**Comment by vanyasem** [comment]
https://t.me/ubports/132418

_Rodney Dawes_
> everything should have a "master" branch that is not tied to a specific ubuntu version (but may require newer versions because of features/dependencies), and all development should happen there. ideally anything in that branch will compile against xenial or bionic. and if there needs to be stable branches for certain things, there can be, and things can be backported as necessary. and then builds of things would happen similar to how we were doing them before, where packages would get built for multiple supported ubuntu series, from the same branch, and only the debian/changelog for the latest series would get merged to master when a build happened
